### PR TITLE
Fix: ssh - Allow keys to sudo user folder

### LINKED ISSUE #715: [BUG] ssh_remote_keys only add keys to root account
BlueBanquise allows being run as a privileged user. 
Some roles now uses the `ansible_env.SUDO_USER` variable to check which user should be impacted with the role.

ssh_remote_keys isn't (yet).

## Possible Fix

in `tasks/main.yml`, replace `user: root` with:
```yaml
- name: authorized_key █ Add authorized key
  ansible.posix.authorized_key:
    user: "{% if ansible_env.SUDO_USER is defined %}/home/{{ ansible_env.SUDO_USER }}{% else %}/root{% endif %}/.ssh"
    key: "{{ item }}"
    exclusive: no
    state: present
  with_items: "{{ authentication_ssh_keys }}"
```

### PR DESCRIPTION
**Pull request by oxedions**
Fix #715 issue
@wlln please check if it is ok for you 😊 
Once merged, 1.6 release.

### CODE REVIEW

File: roles/core/ssh_remote_keys/vars/main.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 ---
-ssh_slave_role_version: 1.1.0
+ssh_slave_role_version: 1.1.1
```
Comment (neilmunday) [inline roles/core/ssh_remote_keys/vars/main.yml:2]: Should this role still have this variable names? Perhaps it should be `ssh_remote_keys_version`?

Comment (strus38) [reply roles/core/ssh_remote_keys/vars/main.yml:2]: +1

**Review by strus38 — CHANGES_REQUESTED** [review]
Should be good to update rile variable name to the role name.

**Comment by wlln** [comment]
Dear @oxedions 

Apart from the point @strus38 and @neilmunday suggest, I'm fine with this PR.

**Comment by oxedions** [comment]
Thanks a lot for the review ! You are right, my bad. Let me fix that 😊

**Pull request merged**